### DISCUSSION TITLE
Can QueryManager Show More than one date query log?

### DISCUSSION THREAD

**Discussion by g-industrialist** [dbeaver/dbeaver#22633] [General]
Once my pc shutdown / my dbeaver shutdown, the query manager clears all query history.  **How can I get back , T- 1, T-2 .... T-N query history ?**

**Comment by LonwoLonwo** [comment]
Hello @iseehk 

DBeaver saves history only in Commercial versions: https://github.com/dbeaver/dbeaver/wiki/Query-Manager

But you can find your history in logs if the setting "save to logs" is checked.
